tests/fixtures/tests.py

```diff
@@ -778,3 +778,17 @@ def test_format_discovery(self):
             '<Article: Time to reform copyright>',
             '<Article: Poker has no place on ESPN>',
         ])
+
+
+class PostgresNulLoadingTests(TestCase):
```

Comment:
I don't think a separate test class isn't needed. I'd put the test  below `test_loaddata_error_message`.